core/id_test.go

```diff
@@ -0,0 +1,11 @@
+package core
+
+import (
+	"testing"
+)
+
+func BenchmarkNextID(b *testing.B) {
```

Comment:
Documenting the results of the testing will be helpful as tomorrow if the performance of the logic which is getting tested gets degraded, we will atleast know about the previous benchmark data and then move in the direction of fixing the issue.